attachments/static/attachments/js/attachments.js

```diff
@@ -15,6 +15,7 @@
                 url: settings.url,
                 success: function(html) {
                     $(settings.container).empty().append(html);
+                    $('#attachments').trigger('table-changed');
```

Comment:
This should probably trigger on `settings.container`, not hard-coded to `#attachments` - in which case, it can probably be chained on the end of the line above.